packages/xs-extra/rrd-transport.master/opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "1.2"
+maintainer: "john.else@citrix.com"
+authors: "john.else@citrix.com"
+homepage: "https://github.com/xapi-project/rrd-transport"
+dev-repo: "https://github.com/xapi-project/rrd-transport"
+bug-reports: "https://github.com/xapi-project/rrd-transport"
+build: [[ "jbuilder" "build" "-p" name "-j" jobs ]]
+build-test: ["jbuilder" "runtest" "-p" name]
+depends: [
+  "jbuilder" {build & >= "1.0+beta10"}
+  "cstruct" {< "3.0.0"}
```

Comment:
We have cstruct 3.1 in the next branch :/

Reply:
We'll fix it for next then :)